src/main/java/io/redisearch/AggregationResult.java

```diff
@@ -11,15 +11,20 @@
  * Created by mnunberg on 2/22/18.
  */
 public class AggregationResult {
-  
- 	private long cursorId = -1;
+
+    public final long totalResults;
```

Comment:
@timmixell can you please add a getting and park it as private?

Reply:
@gkorland I can, but I was using SearchResult for reference (literally copied the lines over). I was going for consistency, but I can push the change up if you'd still like me to proceed.